src/app/pages/home/footer/footer.component.html

```diff
@@ -1 +1,31 @@
-<div>Footer Page</div>
\ No newline at end of file
+
+<nav class=" navbar navbar-dark bg-dark mt-10 clearfix">
+
+    <div class="navbar-expand m-auto navbar-text">
+        Made with
+        <fa-icon [icon] = "heart" class = "hrt" ></fa-icon>
+        by NTUOSS
+        
+    </div>    
+    <div class="navbar-expand m-auto navbar-text ">
+        Follow Us On Social Media
+        <a href= "https://www.facebook.com/ntuoss/?ref=br_rs" target="_blank">
+        <button type = "button" class = "btn">
+            <fa-icon [icon] = "fb"> </fa-icon>      

```

Comment:
Suggest to change the reference to the Facebook icon too as described above.
```suggestion
            <fa-icon [icon]="['fab', 'facebook-f']"></fa-icon>      
```